examples/lab-robust/v2/qwiklabs.es.yaml

```diff
@@ -0,0 +1,39 @@
+# qwiklabs.es.yaml for Spanish localization
+# There is no need to copy everything over from qwiklabs.yaml.
+# Only locale specific attributes need to go in here.
+
+title: Ejemplo de Robust Lab
+description: En serio, el mejor lab que has tomado. Sin excepción.
+
+resources:
+  - type: file
+    id: sample-pdf
+    title: Ejemplo de PDF
+    description: Este PDF contiene todos los ejemplos de código para el laboratorio.
+    uri: resources/sample-es.pdf
+  - type: video
+    id: intro-video
+    title: ¡Bienvenido a GCP!
+    uri: https://www.youtu.be/7jjoyy7_RCk
+    description: Descripción general de Google Cloud Platform
+    duration: 360
+
+environment:
+  student_visible_outputs:
+  - label: Abra la consola de GCP
+  - label: Proyecto GCP
+  - label: Nombre de usuario de GCP
+  - label: Contraseña de GCP
+  - label: InstanceDns
+  - label: Número de cuenta de AWS
+  - label: nombre de usuario
+  - label: clave
+  - label: URL de la consola de AWS
+
+assessment:
+  steps:
+  - title: Crear un depósito de almacenamiento en la nube
+    student_messages:
```

Comment:
think it needs a `locale_id` here for getting assessment/steps translated
https://github.com/Qwiklabs/git-whisperer/blob/c0a62a65a8c217db562d8cba5411427be4f1e76e/whisperer/utils/yaml_merger.py#L125